python/graphstorm/model/embed.py

```diff
@@ -179,6 +179,19 @@ def __init__(self,
         self.dropout = nn.Dropout(dropout)
         self.use_node_embeddings = use_node_embeddings
         self.activation = activation
+
+        # NCCL backend is not supported for utilizing learnable embeddings on nodes. It has a
+        # dependency on distDGL (PR https://github.com/dmlc/dgl/pull/5929).
+        # TODO (Israt): Add NCCL support to distDGL.
+        if th.distributed.is_initialized() and th.distributed.get_backend() == "nccl":
+            if self.use_node_embeddings:
+                raise NotImplementedError('NCCL backend is not supported for utilizing \
+                    node embeddings. Please use gloo backend.')
+            for ntype in g.ntypes:
+                if not feat_size[ntype]:
```

Comment:
```suggestion
                if feat_size[ntype] == 0:
```
This is more easy to understand